admin-console/src/main/webapp/src/components/SystemBehaviorGroups/DeleteBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { ActionGroup, Button, Modal, ModalVariant, Spinner, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { BehaviorGroup } from '../../types/Notifications';
+
+interface DeleteBehaviorGroupModalProps {
+    onDelete: (systemBehaviorGroup?: BehaviorGroup) => Promise<boolean>;
+    bundleName?: string;
+    systemBehaviorGroupName?: string;
+    isOpen: boolean;
+    onClose: () => void;
+
+}
+export const DeleteBehaviorGroupModal: React.FunctionComponent<DeleteBehaviorGroupModalProps> = (props) => {
+    const [ errors, setErrors ] = React.useState(true);
+
+    const onDelete = React.useCallback(async () => {
```

Comment:
I will need to add checks here for unlinking any event-types before deleting.

Reply:
You don't need to do the unlinking - upon deletion, they are unlinked.